### ISSUE TITLE
Code 404: device not found with Alexa

### ISSUE THREAD

**Issue by chrisgrieser** [Rigellute/spotify-tui#1072]
similar to #931 #762, but I am using `spotify-tui` with Alexa instead of `spotifyd`. Worked fine for a good while, but suddenly it's starting to get `404 device not found` constantly, to the point where `spt` is not usable anymore. The interface does popup though and does show the playing information etc.

Tried rebooting, reinstalling, and re-authenticating via spotify

**Comment by sewnie** [comment]
is it recognized by other spotify clients? such as the propietary one and [spotify-player](https://github.com/aome510/spotify-player) ?

**Comment by chrisgrieser** [comment]
works with the webapp (https://open.spotify.com/) and with the official desktop client. Does not work with `spotify-player` (though I am not sure I have set it up correctly, since it only asks for a client-id, but not a client secret?)

**Comment by sewnie** [comment]
> Does not work with spotify-player (though I am not sure I have set it up correctly, since it only asks for a client-id, but not a client secret?)

well, it sort of works on my machine, does it recognize devices like a web client or a phone? if not, open a new issue, maybe your configuration is wrong

**Comment by chrisgrieser** [comment]
it recognizes the phone and the web player.

Also, I already used spotify-tui for a while, it more or less just stopped working yesterday, so I don't think it's an config issue?

**Comment by sewnie** [comment]
yeah, seems to be an upstream spotify api change probably, try reporting it in librespot

**Comment by kingosticks** [comment]
Sorry, I don't understand. This project uses Spotify's web API to select an Alexa device for playback. Where does librespot fit into this scenario?

**Comment by sewnie** [comment]
oops, i thought spotify connect was an integral part of librespot, if this is a issue actually with spotify-tui, i suggest you open an issue on spotify-player, as spotify-tui is literal abandonware.
